onnxruntime/test/contrib_ops/col2im_test.cc

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <stdexcept>
+#include "gtest/gtest.h"
+#include "test/providers/provider_test_utils.h"
+#include "core/util/math.h"
+
+namespace onnxruntime {
+namespace contrib {
+namespace test {
+using namespace onnxruntime::test;
+
+namespace {
+template <typename T>
+std::vector<T> _transpose_serialized_vector(std::vector<T> &input, size_t N, size_t C, size_t H, size_t W) {
```

Comment:
nit: the name with a leading underscore is a bit unconventional here - consider something like "TransposeSerializedVector"

Reply:
done, the intention was to make it "private", but the unnamed namespace did that already